nuxeo-runtime/nuxeo-connect-standalone/src/main/java/org/nuxeo/launcher/connect/ConnectBroker.java

```diff
@@ -1272,10 +1273,16 @@ public boolean pkgRequest(List<String> pkgsToAdd, List<String> pkgsToInstall, Li
                             cmdOk = false;
                         }
                     }
-                    for (String pkgId : localSnapshotsToReplace) {
-                        LocalPackage addedPkg = pkgAdd(pkgId, ignoreMissing);
-                        if (addedPkg == null) {
-                            cmdOk = false;
+                    for (String pkgIdOrFileName : localSnapshotsToReplace) {
+                        if (isLocalPackageFile(pkgIdOrFileName) || isRemotePackageId(pkgIdOrFileName)) {
+                            LocalPackage addedPkg = pkgAdd(pkgIdOrFileName, ignoreMissing);
+                            if (addedPkg == null) {
+                                cmdOk = false;
+                            }
+                        } else {
+                            log.info(String.format(
+                                    "The SNAPSHOT package %s is not available remotely, local cache will be used.",
```

Comment:
checked with karin, my version is good :D, "not remotely available" means "not available at all